docs/guides/jsdom.md

```diff
@@ -17,7 +17,7 @@ As a result, a standalone script like the one below is generally a good approach
 var jsdom = require('jsdom').jsdom;
 
 global.document = jsdom('');
-global.window = document.defaultView;
+global.window = global;
 Object.keys(document.defaultView).forEach((property) => {
```

Comment:
this is an important part of jsdom (using `document.defaultView`).

This loop should ensure that `setTimeout` etc are copied from `document.defaultView` to `global`.